nav_msgs/Path[] triedPaths
geometry_msgs/Vector3[] initialSearch
geometry_msgs/Vector3 oldFrame
int16 chosen
